pkg/training/trainer_pytorch.go

```diff
@@ -344,37 +334,8 @@ func (tt *PyTorchJobTrainer) isPyTorchPod(name, ns string, pod *v1.Pod) bool {
 	return utils.IsPyTorchPod(name, ns, pod)
 }
 
-func (tt *PyTorchJobTrainer) resources(statefulsets []*appv1.StatefulSet, batchJobs []*batchv1.Job, name string, namespace string, pods []*v1.Pod) []Resource {
-	resources := []Resource{}
-	// 2. Find the pod list, and determine the pod of the job
-	for _, sts := range statefulsets {
-		if sts.Labels["pytorch_job_name"] != name {
-			continue
-		}
-		if sts.Namespace != namespace {
-			continue
-		}
-		resources = append(resources, Resource{
-			Name:         sts.Name,
-			Uid:          string(sts.UID),
-			ResourceType: ResourceTypeStatefulSet,
-		})
-	}
-
-	for _, job := range batchJobs {
-		if job.Namespace != namespace {
-			continue
-		}
-		if job.Labels["pytorch_job_name"] != name {
-			continue
-		}
-		resources = append(resources, Resource{
-			Name:         job.Name,
-			Uid:          string(job.UID),
-			ResourceType: ResourceTypeJob,
-		})
-	}
-	resources = append(resources, podResources(pods)...)
+func (tt *PyTorchJobTrainer) resources(name string, namespace string, pods []*v1.Pod) []Resource {
+	resources := podResources(pods)
 	return resources
```

Comment:
How about `return podResources(pods)`?

Reply:
I will try to avoid calling another function in a return statement,  as this approach is more debug-friendly.